components/LeagueEntries/LeagueEntries.tsx

```diff
@@ -16,67 +15,112 @@ import Link from 'next/link';
  * @param props.isEliminated - If true, the user is flagged as eliminat4ed
  * @param props.isPickSet - if true, the team logo of the picked team shows up on the LeagueEntries card and the button changes from "make a pick" to "chagne pick"
  * @param props.teamLogo - the team logo
+ * @param props.lockout - if true, the user is locked out from making a pick
  * @returns {React.JSX.Element} - A div element that contains the user's entry information
  */
+
+/**
+ * Display all entries for a league.
+ * @param {string} leagueId - The league id.
+ * @returns {JSX.Element} The rendered entries component.
+ */
 const LeagueEntries = ({
   entryName,
-  linkUrl,
   isEliminated = false,
   isPickSet = false,
+  linkUrl,
+  isLockedOutProp = false,
   teamLogo = '',
-}: ILeagueEntriesProps): JSX.Element => (
-  <div
-    data-testid="league-entry-container-card"
-    className={cn(
-      'league-entry-container-card grid h-20 min-w-fit grid-cols-2 justify-between rounded-lg border border-border bg-card p-4 text-card-foreground shadow-sm',
-      isEliminated ? 'bg-muted' : 'transparent',
-    )}
-  >
-    <section
-      className="league-entry-header flex items-center gap-12"
-      data-testid="league-entry-header"
+}: ILeagueEntriesProps): JSX.Element => {
+  const [isLockedOut, setLockedOut] = useState<boolean>(isLockedOutProp);
+
+  useEffect(() => {
+    /**
+     * Checks if the user is locked out from making a pick
+     */
+    const checkLockout = (): void => {
```

Comment:
If we rename this function to `checkUserLockout`, would we need this comment?

Reply:
Don't even need the comment, I left that in there as I was working through the logic in my head 😅